system/services/devtools/ScaffoldingService.cfc

```diff
@@ -81,13 +81,13 @@ component singleton=true {
 		filesCreated.append( scaffoldPageTypeForm( pagetype=arguments.id, formName=arguments.id, subDir="page-types", extension=arguments.extension, fields=ListToArray( arguments.fields ) ) );
 
 
-		i18nProps[ "name" ]        = arguments.name;
-		i18nProps[ "description" ] = arguments.description;
+		i18nProps[ "name" ]        = _convertToReadableFormat( words = arguments.name );
```

Comment:
I think we should only be converting inputs that are field names. Descriptions and name's are already input by users as free text, not constrained. Places I believe we should convert:

1. Field names in preside objects
2. Field names in system config forms (`new configform` command)
3. Field names in widget config forms (`new widget` command)
4. Field names in page types (`new pagetype` command)

Reply:
Nice catch there. Totally overlooked the names and descriptions as free text. Changes now committed and pushed.